### PR TITLE
Logmatic handler

### PR DESCRIPTION
**Pull request by gpolaert**
Add the Logmatic.io handler, extram from the master branch.

### CODE REVIEW

File: src/Monolog/Formatter/LogmaticFormatter.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    /**
+     * @param string
+     */
+    protected $hostname = '';
+
+    /**
+     * @param string
+     */
+    protected $appname = '';
```
Comment (stof) [inline src/Monolog/Formatter/LogmaticFormatter.php]: I would make these private to simplify the handling of backward compatibility

File: src/Monolog/Handler/LogmaticHandler.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    /**
+     * @var string
+     */
+    protected $hostname;
+
+    /**
+     * @var string
+     */
+    protected $appname;
```
Comment (stof) [inline src/Monolog/Handler/LogmaticHandler.php]: same here

File: tests/Monolog/Handler/LogmaticHandlerTest.php [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    }
+
+    private function createHandler()
+    {
+        $useSSL = extension_loaded('openssl');
+        $args = array('testToken', 'testHostname', 'testAppname', $useSSL, Logger::DEBUG, true);
+        $this->res = fopen('php://memory', 'a');
+        $this->handler = $this->getMockBuilder('Monolog\Handler\LogmaticHandler')
+            ->setConstructorArgs($args)
+            ->setMethods(['fsockopen', 'streamSetTimeout', 'closeSocket'])
```
Comment (stof) [inline tests/Monolog/Handler/LogmaticHandlerTest.php]: you still have a short array syntax here

File: src/Monolog/Handler/LogmaticHandler.php (hunk 2 above)
Comment (gpolaert) [inline src/Monolog/Handler/LogmaticHandler.php]: Same behavior the hostname prop?

**Comment by Seldaek** [comment]
Sorry but rather not add more handlers to 1.x at this point.. It's in master and I'll try to wrap up 2.0 so it becomes usable in a stable version :)

**Pull request closed without merging**